lib/inc/drogon/drogon_test.h

```diff
@@ -927,7 +927,27 @@ static int run(int argc, char** argv)
         drogon::test::internal::numCorrectAssertions++; \
     } while (0)
 
-#define FAIL(message)                                                   \
+#ifdef __cpp_fold_expressions
+template <typename... Args>
+inline void outputReason(Args&&... args)
+{
+    (std::cout << ... << std::forward<Args>(args));
+}
+#else
+template <typename Head>
+inline void outputReason(Head&& head)
+{
+    std::cout << std::forward<Head>(head);
+}
+template <typename Head, typename... Tail>
+inline void outputReason(Head&& head, Tail&&... tail)
+{
+    std::cout << std::forward<Head>(head);
+    outputReason(std::forward<Tail>(tail)...);
+}
+#endif
```

Comment:
Please put functions that is only used by the framework itself inside a `internal` namespace.

Reply:
I totally forgot to wrap them into namespaces, thank you for your review!